### PR TITLE
JBPM-5707: Uploaded java file should be placed under src/main/java

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by manstis** [comment]
Woot :-)

**Comment by wmedvede** [comment]
and yes ... @manstis would you mind review ;)

File: guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ProjectResourcePathResolver.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ * limitations under the License.
+ */
+
+package org.guvnor.common.services.project.backend.server;
+
+import org.guvnor.common.services.project.model.Package;
+import org.uberfire.backend.vfs.Path;
+
+/**
+ * Resolves the by default target path withing a project for a given resource based on it's type.
```
Comment (manstis) [inline guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ProjectResourcePathResolver.java]: Typo "within**g**" s/b "within"

**Comment by wmedvede** [comment]
@manstis typo fix in place.
Thanks!

**Pull request merged**